SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java

```diff
@@ -0,0 +1,105 @@
+package com.morse.selectchannel;
+
+import android.os.Bundle;
+
+import com.google.android.material.appbar.CollapsingToolbarLayout;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.google.android.material.snackbar.Snackbar;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.view.View;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class SelectChannelActivity extends AppCompatActivity {
+    ListView listView;
+    String mTitle[] = {"Messenger", "Whatsapp", "Twitter", "Instagram"};
+    String mDescription[] = {"Canalul pentru Messenger", "Canalul pentru Whatsapp", "Canalul pentru Twitter", "Canalul pentru Instagram"};
```

Comment:
Change these to English, for consistency.

Reply:
Yep, did it.